File: medplum-java/medplum-model/src/main/java/com/fhir/EffectEvidenceSynthesis_Certainty.java
package com.fhir;

import com.StagedBuilderStyle;
import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
import java.lang.String;
import java.util.List;
import java.util.Optional;
import org.immutables.value.Value;

@Value.Immutable
@StagedBuilderStyle
@JsonDeserialize(
    as = ImmutableEffectEvidenceSynthesis_Certainty.class
)
@JsonIgnoreProperties(
    ignoreUnknown = true
)
public interface EffectEvidenceSynthesis_Certainty {
  Optional<List<Extension>> modifierExtension();

  Optional<List<EffectEvidenceSynthesis_CertaintySubcomponent>> certaintySubcomponent();

  Optional<List<Extension>> extension();

  Optional<List<CodeableConcept>> rating();

  Optional<String> id();

  Optional<List<Annotation>> note();

  static ImmutableEffectEvidenceSynthesis_Certainty.Builder builder() {
    return ImmutableEffectEvidenceSynthesis_Certainty.builder();
  }
}

File: medplum-typescript/packages/sdk/src/model/fhir/EffectEvidenceSynthesis_Certainty.ts
import { Annotation } from "./Annotation";
import { CodeableConcept } from "./CodeableConcept";
import { EffectEvidenceSynthesis_CertaintySubcomponent } from "./EffectEvidenceSynthesis_CertaintySubcomponent";
import { Extension } from "./Extension";

export interface EffectEvidenceSynthesis_Certainty {
    id: string | null | undefined;
    extension: Extension[] | null | undefined;
    modifierExtension: Extension[] | null | undefined;
    rating: CodeableConcept[] | null | undefined;
    note: Annotation[] | null | undefined;
    certaintySubcomponent:
        | EffectEvidenceSynthesis_CertaintySubcomponent[]
        | null
        | undefined;
}
